google-cloud-spanner/lib/google/cloud/spanner/convert.rb

```diff
@@ -223,6 +222,8 @@ def grpc_value_to_object value, type
               Data.from_grpc value.list_value.values, type.struct_type.fields
             when :NUMERIC
               BigDecimal value.string_value
+            when :JSON
+              JSON.parse value.string_value unless value.string_value.empty?
```

Comment:
Do we need `unless value.string_value.empty?`? If `string_value` is empty, it should throw an error, right? E.g., looking at `timestamp`, `date`, and `numeric`.

Reply:
done